### DISCUSSION TITLE
[Geochemistry module][Coupling of transportation and reactions]

### DISCUSSION THREAD

**Discussion by coskrrb2002** [idaholab/moose#20871] [Q&A Modules: General] [answered]
Dear users,

I would like to use geochemistry module for coupling of transportation and chemical reactions.

As far as I know, there are three ways to couple the transportation and chemical reactions.
1.  Fully coupling
2.  Solve transportation -> Solve chemical reactions.
3.  Itration of Solve transporation -> Solve chemical reactions

Among these three types, which method is used in the Geochemistry module?

Cheers,
Q

**Comment by WilkAndy** [comment, marked as answer]
The answer is (2) and (3).   You might find the quite detailed documentation found at https://mooseframework.inl.gov/modules/geochemistry/index.html useful.

**Reply by coskrrb2002** [reply]
Thank you for your answer! Its great helpful for me~! Cheers
